### PR TITLE
PAQ change log entry for c8y_Position (PAM-34683)

### PR DESCRIPTION
**Pull request by kailol-sag**

### CODE REVIEW

File: content/change-logs/analytics/apama-in-c8y-20240425-New-EPL-actions-for-creating-and-parsing-the-c8y_Position-fragment.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+component:
+  - value: component-M5-cepIIS
+    label: Streaming Analytics
+build_artifact:
+  - value: tc-KXXmo2SUR
+    label: apama-in-c8y
+ticket: PAM-34615
+version:
+---
+The `ManagedObject` event has a unique way of handling `c8y_Position` fragments that have the parseable `float`  values added to their position property, and all other values added to their `params` property. All other events of the Cumulocity IoT transport API simply add the fragment as a whole to their `params` property. This has made it cumbersome to robustly copy `c8y_Position` to a `ManagedObject` from another event and vice versa. To make this easier, two new helper actions have been added to the `ManagedObject` event to populate `ManagedObject` from a `c8y_Position` object, and conversely create the object from the `ManagedObject` event.
```
Comment (Rob-J-SAG) [inline content/change-logs/analytics/apama-in-c8y-20240425-New-EPL-actions-for-creating-and-parsing-the-c8y_Position-fragment.md:17]: Double space after 'float'.

**Review by Rob-J-SAG — APPROVED** [review]

**Review by nmfa — APPROVED** [review]
Looks good.

**Review by MWindel — APPROVED** [review]

**Comment by skom-sag** [comment]
Part of PAM 10.15.5.0.561255

**Comment by skom-sag** [comment]
PAQ 25.137.0 is deployed to zone-1 / eu-latest today

**Pull request merged**